core/src/apps/webauthn/fido_seed.py

```
@@ -0,0 +1,18 @@
+def ensure_fido_seed(func):
+    def wrapper(*args, **kwargs):
```

Ruff (0.8.2)

Missing return type annotation for private function `wrapper`

(ANN202)